networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -244,8 +245,8 @@ TEST_F(CentralityGTest, testKatzDynamicAddition) {
     DEBUG("finish kc");
     node u, v;
     do {
-        u = GraphTools::randomNode(G);
-        v = GraphTools::randomNode(G);
+        u = G.randomNode();
+        v = G.randomNode();
```

Comment:
Random unrelated changes to the PR. Please remove this.

Reply:
That was a mistake, I removed this change.